docs: inside-arbitrum-nitro (#3349)
OffchainLabs/arbitrum-docs: docs/partials/glossary/_l2.mdx

---
title: L2

> key: l2

Reviewer: ```suggestion
key: L2
```

titleforSort: L2
---

> An L2, or Layer 2, is a trustless scaling solution built on top of Ethereum's Layer 1 (L1) base protocol.

Reviewer: ```suggestion
An L2, or Layer 2, is a trustless scaling solution built on top of Ethereum's <a data-quicklook-from="layer-1-l1">Layer 1 (L1)</a> base protocol.
```

> Arbitrum is a Layer 2 platform that increases throughput and reduces the cost of transactions on Ethereum's (L1) without introducing additional trust assumptions. It does that by executing some computation offchain and batch-posting transactions on the underlying Layer 1 chain (Ethereum for Arbitrum One/Nova).

Reviewer: ```suggestion
<a data-quicklook-from="arbitrum">Arbitrum</a> is a Layer 2 platform that increases throughput and reduces the cost of transactions on Ethereum (L1) without introducing additional trust assumptions. It does that by executing some computation offchain and batch-posting transactions on the underlying Layer 1 chain (Ethereum for Arbitrum One/Nova).
```
